packages/chart/src/components/EditorPanel/EditorPanel.tsx

```diff
@@ -2702,6 +2702,17 @@ const EditorPanel = () => {
                                 />
                               </label>
 
+                              <label>
```

Comment:
Shouldn't this be set by the filter value itself? Like wouldn't the filter have already responded to the query param change?

Reply:
I'm not sure I understand what you're asking. This section of the code is so that the user can specify in the editor which QS param they want to map to which filter dropdown

Reply:
I see now that you can set filters directly in the chart editor. that's what I was confused about.